pysilcam/tracking/track.py

```diff
@@ -0,0 +1,417 @@
+from docopt import docopt
+from pysilcam.tracking.silcamtracker import *
```

Comment:
F403: 'from pysilcam.tracking.silcamtracker import *' used; unable to detect undefined names  <!-- Automatically posted by Lintly -->